transport/processing/processing_model.py

```diff
@@ -0,0 +1,7 @@
+import collections
+
+ProcessingJob = collections.namedtuple(
+    'ProcessingJob',
+    ['component', 'container_image', 'download_request',
```

Comment:
codestyl-nit: in our codebas, we prefer to have a single item per line, or all in the same line (not mix styles)
also, consider using @dataclasses.dataclass - those are (I think) easier to read, while introducing minimal boilerplate overhead (and allow for more flexibilty)